static/layout.less

```diff
@@ -1,4 +1,5 @@
 #everything {
+  -js-display: flex; // ie9/10 flexbox fix
```

Comment:
Could this be added to the `.flex` mixin?

Reply:
Great idea. Done!